packages/uniforms/src/BaseForm.js

```diff
@@ -80,7 +82,13 @@ export default class BaseForm extends Component {
     constructor () {
         super(...arguments);
 
-        this.state = {bridge: createSchemaBridge(this.props.schema), changed: null, changedMap: {}, resetCount: 0};
+        this.state = {
+            bridge: createSchemaBridge(this.props.schema),
+            changed: null,
+            changedMap: {},
+            submitting: null,
```

Comment:
I think we can start with `false` here - `changed` is somehow different, as it's used to determine the first render.